beacon-chain/rpc/apimiddleware/structs.go

```diff
@@ -692,6 +692,10 @@ type BLSToExecutionChangeJson struct {
 	ToExecutionAddress string `json:"to_execution_address" hex:"true"`
 }
 
+type BroadcastBLSToExecutionChangesRequest struct {
+	Data string `json:"data"`
+}
```

Comment:
Maybe let's create an empty struct that can be reused whenever we want to POST without data.
```
type EmptyRequest struct {}
```